policy-controller/grpc/src/outbound.rs

```diff
@@ -41,16 +41,33 @@ where
         OutboundPoliciesServer::new(self)
     }
 
-    fn lookup(
-        &self,
-        spec: outbound::TrafficSpec,
-    ) -> Result<(String, String, NonZeroU16), tonic::Status> {
+    fn lookup(&self, spec: outbound::TrafficSpec) -> Result<OutboundDiscoverTarget, tonic::Status> {
```

Comment:
`lookup_authority` is just a helper used by `lookup`.  It returns the name, namespace, and port for a given authority.  There wouldn't really be any benefit to passing in the TrafficSpec or source workload just so that it could pass it back out as part of an OutboundDiscoverTarget.